src/modules/vtol_att_control/tiltrotor.cpp

```diff
@@ -207,8 +207,11 @@ void Tiltrotor::update_mc_state()
 {
 	VtolType::update_mc_state();
 
-	// make sure motors are not tilted
-	_tilt_control = _params_tiltrotor.tilt_mc;
+	VtolType::pusher_assist();
+
+	_tilt_control = _forward_thrust;
```

Comment:
@sfuhrer Details, but would it make sense to have pusher_assist() return forward_thrust?

Reply:
true, I've changed that.